### PR TITLE
AirfRANS 4L/256d gc=1.5 + T_max=10 — wider cosine cycles

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis

At the 3L/192d scale, gc=1.5 beat gc=1.0 significantly (0.00935 vs 0.01419). The 4L/256d golden config uses gc=1.0 with T_max=5 (5-epoch cosine cycles). We hypothesise that at the 4L/256d scale, slightly more permissive gradient clipping (gc=1.5) combined with longer cosine cycles (T_max=10) will allow the model to explore wider loss basins and find a better minimum. The two changes are tested together as they are complementary: wider cycles benefit from slightly larger gradient steps.

## Instructions

Run the 4L/256d config with gc=1.5 and T_max=10:

```bash
cd target/icml2026 && SENPAI_MAX_EPOCHS=9999 python train.py \
  --dataset airfrans --airfrans_task full --optimizer adamw \
  --lr 7e-4 --cosine-t-max 10 --grad-clip 1.5 --weight-decay 1e-2 \
  --no-use-ema --enable-fourier \
  --model-layers 4 --model-hidden-dim 256 --model-heads 4 \
  --epochs 999 \
  --wandb_group airfrans-4L256d-gc15
```

Key differences from baseline: `--grad-clip 1.5` (was 1.0) and `--cosine-t-max 10` (was 5)

## Baseline

Current best (PR #2727, W&B run: ruurxdqs):
- val_primary/surface_mse: **0.007264** (at epoch 40, hit cap at epoch 50)
- External target: **0.0043**
- Baseline W&B run: [ruurxdqs](https://wandb.ai/wandb/senpai/runs/ruurxdqs)
- Reproduce: `cd target/icml2026 && SENPAI_MAX_EPOCHS=9999 python train.py --dataset airfrans --airfrans_task full --optimizer adamw --lr 7e-4 --cosine-t-max 5 --grad-clip 1.0 --weight-decay 1e-2 --no-use-ema --enable-fourier --model-layers 4 --model-hidden-dim 256 --model-heads 4 --epochs 999`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2762 because ADVISOR: Closing — gc=1.5 is confirmed dead at 4L/256d (5 independent confirmations: haku, kohaku, armin, fern all failed). Deeper architectures amplify gradients making gc=1.5 harmful..
